Swin-Transformer/Swin-Transformer.patch

```diff
@@ -71,7 +71,16 @@ index 84230ea..faa9291 100644
  
  def parse_option():
      parser = argparse.ArgumentParser('Swin Transformer training and evaluation script', add_help=False)
-@@ -73,6 +78,12 @@ def parse_option():
+@@ -64,7 +69,7 @@ def parse_option():
+     parser.add_argument('--throughput', action='store_true', help='Test throughput only')
+ 
+     # distributed training
+-    parser.add_argument("--local_rank", type=int, required=True, help='local rank for DistributedDataParallel')
++    parser.add_argument("--local_rank", type=int, help='local rank for DistributedDataParallel')
+ 
+     # for acceleration
+     parser.add_argument('--fused_window_process', action='store_true',
```

Comment:
Why add this line in patch?

Reply:
You mean line 79?  It is not a new line, just remove `required=True`.

Reply:
I mean line 82

Reply:
There is no '+' before line 82, means there is no change to this line. The patch is generated from `git diff`, I guess it adds the context of the changed line.

Reply:
Got it.